Composer/packages/server/src/models/bot/botProject.ts

```diff
@@ -488,6 +489,46 @@ export class BotProject implements IBotProject {
     return qnaEndpointKey;
   };
 
+  public async generateDialog(name: string) {
+    const defaultLocale = this.settings?.defaultLanguage || defaultLanguage;
+    const relativePath = defaultFilePath(this.name, defaultLocale, `${name}${FileExtensions.FormDialogSchema}`);
+    const schemaPath = Path.resolve(this.dir, relativePath);
+
+    const dialogPath = defaultFilePath(this.name, defaultLocale, `${name}${FileExtensions.Dialog}`);
+    const outDir = Path.dirname(Path.resolve(this.dir, dialogPath));
+
+    const feedback = (type: FeedbackType, message: string): void => {
+      // eslint-disable-next-line no-console
+      console.log(`${type} - ${message}`);
+    };
+
```

Comment:
Is there a link that describes this giant list of parameters? Its a long list!
If not can we add some inline documentation here to explain them all?

Reply:
The experimental library has documentation as part of being on the command line, but it changes a lot.  Good point - I will add comments here too.